config/application.example.yml

```diff
@@ -0,0 +1,6 @@
+gmail_username: 'no-reply@gmail.com'
+
+production:
+  gmail_username: <%= ENV["gmail_username"] %> 
```

Comment:
No, we shouldn't do that.
Think about it
```yml
test:
  some_message: 'Hello world'
```

Then in the app we can use it like `ENV['some_message']`

And I think for production we shouldn't write the value. Leave them blank.